### PR TITLE
Add HTMLPictureElement Interface

### PR DESCRIPTION
**Pull request by YuichiNukiyama**
fix [HTMLPictureElement interface definition is missing
#5323](https://github.com/Microsoft/TypeScript/issues/5323) issue. And add
 methods of Document & Element interface which related HTMLPictureElement.

### CODE REVIEW

File: baselines/dom.generated.d.ts [hunk 1]
```diff
@@ -2569,6 +2569,8 @@ interface Document extends Node, GlobalEventHandlers, NodeSelector, DocumentEven
       * @param content The text and HTML tags to write.
       */
     writeln(...content: string[]): void;
+    createElement(tagName: "picture"): HTMLPictureElement;
```
Comment (mhegazy) [inline baselines/dom.generated.d.ts]: these will be ignored if they are defined here after the less specific signature: `createElement(tagName: string)`. 
@zhengbli why is the order messed up here? and how can we fix it?

File: baselines/dom.generated.d.ts [hunk 2]
```diff
@@ -2569,6 +2569,8 @@ interface Document extends Node, GlobalEventHandlers, NodeSelector, DocumentEven
...
+    getElementsByTagName(tagname: "picture"): NodeListOf<HTMLPictureElement>;
```
Comment (mhegazy) [inline baselines/dom.generated.d.ts]: same as above.

File: baselines/dom.generated.d.ts [hunk 3]
```diff
@@ -2981,6 +2983,7 @@ interface Element extends Node, GlobalEventHandlers, ElementTraversal, NodeSelec
     webkitRequestFullscreen(): void;
     getElementsByClassName(classNames: string): NodeListOf<Element>;
     matches(selector: string): boolean;
+    getElementsByTagName(tagname: "picture"): NodeListOf<HTMLPictureElement>;
```
Comment (mhegazy) [inline baselines/dom.generated.d.ts]: and here too.

File: baselines/dom.generated.d.ts (hunk 1 above)
Comment (zhengbli) [inline baselines/dom.generated.d.ts]: Actually the overloads wouldn't be ignored, even if they are defined after the more generic signature. The reason why the order is messed up is that the information in the json files are picked up after the emitting from the spec was done, so they are always placed at the end. I can fix it later, though it shouldn't affect the functionality now.

**Comment by zhengbli** [comment]
LGTM.

**Pull request merged**